cmd/operator/run.go

```diff
@@ -26,8 +26,8 @@ import (
 	"os"
 	"time"
 
+	crddefaults "github.com/blackducksoftware/synopsys-operator/cmd/synopsysctl/cmd"
```

Comment:
Hi @mphammer 

Can we move this file inside pkg/defaults or some name that makes sense to you?

cmd/synopsysctl/cmd is not a good practice.